backends/credhub/src/test/java/org/cloudfoundry/credhub/integration/v1/credentials/CredentialsTypeSpecificSetIntegrationTest.java

```diff
@@ -93,29 +93,71 @@ public class CredentialsTypeSpecificSetIntegrationTest {
     .build();
   private static final String VALUE_VALUE = "test-value";
   private static final String PASSWORD_VALUE = "test-password";
-  private static final String CERTIFICATE_VALUE_JSON_STRING = JSONObject.toJSONString(
-    ImmutableMap.<String, String>builder()
-      .put("ca", TestConstants.TEST_CA)
-      .put("certificate", TestConstants.TEST_CERTIFICATE)
-      .put("private_key", TestConstants.TEST_PRIVATE_KEY)
-      .build());
-  private static final String SSH_VALUE_JSON_STRING = JSONObject.toJSONString(
-    ImmutableMap.<String, String>builder()
-      .put("public_key", TestConstants.SSH_PUBLIC_KEY_4096_WITH_COMMENT)
-      .put("private_key", TestConstants.PRIVATE_KEY_4096)
-      .build());
-  private static final String RSA_VALUE_JSON_STRING = JSONObject.toJSONString(
-    ImmutableMap.<String, String>builder()
-      .put("public_key", TestConstants.RSA_PUBLIC_KEY_4096)
-      .put("private_key", TestConstants.PRIVATE_KEY_4096)
-      .build());
-  private static final String JSON_VALUE_JSON_STRING = JSONObject.toJSONString(jsonValueMap);
+
+  private static final String CERTIFICATE_VALUE_JSON_STRING;
+  static {
+    try {
+      CERTIFICATE_VALUE_JSON_STRING = new ObjectMapper().writeValueAsString(
+              ImmutableMap.<String, String>builder()
+                      .put("ca", TestConstants.TEST_CA)
+                      .put("certificate", TestConstants.TEST_CERTIFICATE)
+                      .put("private_key", TestConstants.TEST_PRIVATE_KEY)
+                      .build()
+      );
+    } catch (JsonProcessingException e) {
```

Comment:
Why bother to catch and rethrow an exception in test code? Is there no way to declare a checked exception in a static code block like this?

Reply:
The IDE shows an error without the try/catch block, as `ObjectMapper().writeValueAsString` could potentially throw an exception. I'm not familiar with "declare a checked exception," could you show an example / docs?

Reply:
I mean using the "throws" keyword. It's a common code smell when we catch an exception when we could just declare it and let it be thrown.

However, it looks like the compiler steered you right in this case. Seems like you cannot use `throws` in static blocks, and the only allowable exception is a runtime exception.